src/server/routes.rs

```diff
@@ -1,9 +1,16 @@
 use std::sync::Arc;
 
+use serde::{Deserialize, Serialize};
 use warp::Filter;
 
 use crate::{server::filters, signature::KeyRing};
 
+#[derive(Deserialize, Serialize)]
+pub struct HealthStatus {
```

Comment:
Is this a type that we expect people to read back from the server using a client? If so, would it be better to put into `invoice/api.rs` or another location?

Reply:
Moved it to `invoice/api.rs` and also changed the name to `HealthResponse`.